jsettlers.logic/src/main/java/jsettlers/ai/economy/BuildingListEconomyMinister.java

```diff
@@ -54,19 +54,19 @@
 	private boolean isMiddleGoodsGame;
 
 	static {
-		EBuildingType[] buildingIndustry = {LUMBERJACK, FORESTER, SAWMILL, STONECUTTER};
+		EBuildingType[] buildingIndustry = { LUMBERJACK, FORESTER, SAWMILL, STONECUTTER };
 		BUILDING_INDUSTRY = Arrays.asList(buildingIndustry);
```

Comment:
Use a Set<EBuildingType> here, since the list is not ordered.

Hint: The EnumSet class provides you with really fast enum sets.